grpc/swift-cubic/CubicManager.swift

```diff
@@ -121,6 +139,7 @@ public class CubicManager: NSObject, AVAudioRecorderDelegate {
         self.startAudioEngine()
     }
     
+    @available(iOS 9.0, *)
     func startAudioEngine() {
```

Comment:
As it currently is implemented, this file should not be in the grpc/swift-cubic directory. The main reason is that it is making some assumptions that are very application specific, like this `startAudioEngine()` function. It is making the assumption that audio is coming from AVAudioEngine, which is not always true (audio might be coming from a file, for example). The grpc/swift-cubic directory should only have methods to allow an SDK user to set up a client and interact with the server. Other logic should be in the examples directory.

Reply:
Based on what we discussed with Julie @renaissanceGeek , we decided to include CubicManager logic into SDK. Should I move working with audio streams to examples and leave here only client setup?